aries-site/src/pages/templates/app-layouts.mdx

```diff
@@ -0,0 +1,170 @@
+relate to each other, the spacing between each, and how they respond 
+at various screen widths.
+
+2. **Global Header & Footer**  
+The Global Header, also known as the HPE Common Header and Footer 
+Web Service, provides a tunable, yet consistent header and footer 
+to be used across all HPE applications. For more information, see 
+[Global Header](/templates/global-header).
+
+3. **Page Container**  
+The Page Container contains the Page Content and the Page Header. 
+It is responsible for the layout for the main content of a page. 
+
+--INSERT-EXAMPLE-HERE--
+
+Jump to:
+- [Container Width](#container-width)
+- [Grid](#grid)
+- [Breakpoints](#breakpoints)
+
+Coming soon:
+- Page Content layout templates for Dashboards, Wizards, Lists and Tables.
+- Updated guidance on [Page Headers](/components/header#page-header).
+
+## Responsive Layouts
+App layouts should be designed to be adaptable to different screen sizes 
+depending on the user input. Our goal, as designers and developers at HPE, 
+should be to create optimal and accessible user experiences regardless of 
+screen size or resolution.
+
+## Page Container
+The Page Container is responsible for the layout for the main content of a page.
+It contains a page's [Page Header](/components/header#header-for-a-single-page) 
+and its Page Content.
+
+### Container Width
+There are 3 content width options from which to choose, depending on 
+the layout use case: [Wide](#wide-default), [Narrow](#narrow) and [Full](#full). 
+
+In all cases, the minimum width of content is `336px`, after which horizontal 
+scrolling is enabled.
+
+#### Wide (default)
+This is the default width for the Page Container. Using the "wide" content width 
+scales the Page Container until a maximum width of `1608px`. If the screen is 
+larger than `1608px`, the Page Container will be center aligned and white space 
+will be displayed around the perimeter of the content. 
+
+--INSERT-EXAMPLE-HERE--
+
+Use cases include:
+- Dashboards 
+- Tables 
+- Lists
+- Charts
+
+--INSERT-EXAMPLE-HERE-?????-TBD--
+
+--INSERT-EXAMPLE-HERE-?????-TBD--
+
+#### Narrow
+The "narrow" option scales the content width until a maximum width of `768px`. 
+If the screen is larger than 768, the Page Container will be center aligned 
+and white space will be displayed around the perimeter of the content. 
+
+--INSERT-EXAMPLE-HERE--
+
+Use cases include:
+- Wizards 
+- Articles 
+- Single task flows, such as forms
+- Admin and user settings
+
+--INSERT-EXAMPLE-HERE-?????-TBD--
+
+--INSERT-EXAMPLE-HERE-?????-TBD--
+
+#### Full
+This option will automatically resize the container to fill the 
+screen displaying it. With the "full" option, if the screen grows wider 
+than the available content, the Page Container will be left aligned to 
+the edge of the screen.
+
+--INSERT-EXAMPLE-HERE--
+
+Use cases include:
+- Charts 
+- Maps 
+- Tables
+
+--INSERT-EXAMPLE-HERE-?????-TBD--
+
+--INSERT-EXAMPLE-HERE-?????-TBD--
+
+### Grid
```

Comment:
We mention the capabilities of `grid` in grommet so should we link out to the props specifically in grommet or to the grid component page in design-system.

Reply:
I like this thought. 

Still debating the best way to present this content and how it unfolds. Right now, this reference to "grid" is the concept of CSS grid and its capabilities. When we start referencing Grommet's Grid, we will definitely want to provide cross-references and documentation.

Reply:
Let's keep this in mind as this page progresses. I looked at our Grid page and am not loving its current content... so thinking if we should try to enhance it along-side this work.